ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -790,6 +786,13 @@ public class BottomSheetController: UIViewController {
             guard let strongSelf = self else {
                 return
             }
+
+            // It's important we drop the reference to the animator as early as possible.
+            // Otherwise we could accidentally try modifying the animator while it's calling out to its completion handler, which can lead to a crash.
+            if let animator = strongSelf.currentStateChangeAnimator,animator == translationAnimator {
```

Comment:
This looks like the CI linter error
```suggestion
            if let animator = strongSelf.currentStateChangeAnimator, animator == translationAnimator {
```